### DISCUSSION TITLE
Moyens de paiement

### DISCUSSION THREAD

**Discussion by pamls** [PayGreen/documentation#9] [Q&A]
Bonjour,

Plusieurs méthodes de paiement sont marqués "upcoming" dans la documentation, comme Monizze, Benefiz et wedofood pourtant leur logo est affiché sur le site paygreen.io et surtout il me semblait qu'il s'agissait de paiement par carte Mastercard.

Est-ce que les clients peuvent utiliser ces 3 moyens de paiement sur paygreen en passant par le module carte bancaire ?

Merci

**Comment by RenaudGreen** [comment]
Bonjour,

Ce onglet est réservé aux questions purement techniques. Le sujet abordé dans votre demande nécessite de passer par le helpdesk https://paygreen.atlassian.net/servicedesk/customer/portal/9 

Merci
